src/services/userService.js

```diff
@@ -14,25 +14,55 @@ const getUserSignUp = async (
   gender,
   password,
   profileImg,
-  age
+  age,
+  admin
 ) => {
   validateId(userId);
   validateNickname(nickname);
   validatePassword(password);
 
   const saltRounds = 10;
   const hashedPw = await bcrypt.hash(password, saltRounds);
-
+  
   return await userDao.createUser(
     userId,
     nickname,
     gender,
     hashedPw,
     profileImg,
-    age
+    age,
+    admin
   );
 };
 
+const signIn = async ( userId, password ) => {
+  const user = await userDao.signIn(userId);
```

Comment:
signIn은 business logic을 의미합니다.
Model Layer (Dao)에서는 data 관련 작업을 의미하는 함수명을 적어주는 것이 좋습니다.

```suggestion
  const user = await userDao.getUserById(userId);
```

Reply:
제시해주신 내용으로 수정했습니다